docs/SETTINGS.rst

```diff
@@ -38,6 +38,11 @@ Dynamic configuration is stored in the DCS (Distributed Configuration Store) and
         -  **type**: slot type. Could be ``physical`` or ``logical``. If the slot is logical, you have to additionally define ``database`` and ``plugin``.
         -  **database**: the database name where logical slots should be created.
         -  **plugin**: the plugin name for the logical slot.
+-  **ignore_slots**: list of sets of replication slot properties for which Patroni should ignore matching slots. This configuration/feature/etc. is useful when some replication slots are managed outside of Patroni. Any subset of matching properties will cause a slot to be ignored.
+    -  **name**: the name of the replication slot.
+    -  **type**: slot type. Can be ``physical`` or ``logical``. If the slot is logical, you may additionally define ``database`` and/or ``plugin``.
+    -  **database**: the database name (when matching a ``logical`` slot).
+    -  **plugin**: the logical decoding plugin (when matching a ``logical`` slot).
```

Comment:
`slots` is a dict, while `ignore_slots` is a list.
What do you think about adding the YAML code snippet with the example, two slots and two ignore_slots? IMO it would help to avoid confusion.

Reply:
Added an example chunk of YAML.

Reply:
For ignored slots, why does one need the extensive config? isn't `name` enough?

Reply:
It's potentially useful (this is in our use case) to exclude on other combinations of config options without having to specify the specific slot. For example, we want to be able to specify that all logical slots using a specific output plugin (where we know the plugin manages the slots) should be ignored.

Reply:
Ah did not get that smart "matching" from the example.